python/scripts/Exp_Dorn2011.py

```diff
@@ -96,14 +96,12 @@
             ca = tm1.charge_accumulation(fr, ecm)
             sn = tm1.stationary_nonlinearity(ca)
             sr = tm1.slow_response(sn)
-#            sr.data=sr.data[0:int(np.round((moviedur+1)/sr.tsample))]
             intfunc= interpolate.interp1d(np.linspace(0, len(sr.data),len(sr.data)),
                                       sr.data)
-            
+            sr_rs=intfunc(np.linspace(0, len(sr.data), len(sr.data)*sr.tsample*fps))
```

Comment:
Yeah - this one was the one I was looking for! Otherwise, I wasn't sure what `sr_rs` would be below. 

Merging!